tasks/jira/src/main/java/com/walmartlabs/concord/plugins/jira/v2/JiraTaskV2.java

```diff
@@ -28,24 +28,33 @@
 
 import javax.inject.Inject;
 import javax.inject.Named;
+import java.util.HashMap;
```

Comment:
```suggestion
```

Reply:
HashMap is still instantiated elsewhere in that class: https://github.com/walmartlabs/concord-plugins/pull/192/files#diff-9d89a607e0ffd525db074ed739a7cea767bbd80daa30e79fbfca613d126c2dceR53